webapp/components/survey/CategoryDetails/store/actions/item/useUpdateItemProp.js

```diff
@@ -0,0 +1,41 @@
+import { useCallback } from 'react'
+import { useDispatch } from 'react-redux'
+import axios from 'axios'
+
+import { AppSavingActions } from '@webapp/store/app'
+import { SurveyActions, useSurveyId } from '@webapp/store/survey'
+
+import { debounceAction } from '@webapp/utils/reduxUtils'
+import { State } from '../../state'
+
+const _putProp = ({ surveyId, categoryUuid, itemUuid, key, value, setState, dispatch }) => {
```

Comment:
_putProp can be the action itself and be invoked with debounce in useUpdateItemProp

Reply:
ok